fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -2854,12 +2853,14 @@ private List<Long> chosenBackendIdBySeq(int replicationNum, TStorageMedium stora
     private List<Long> chosenBackendIdBySeq(int replicationNum) throws DdlException {
         List<Long> chosenBackendIds =
                 systemInfoService.seqChooseBackendIds(replicationNum, true, true);
-        if (CollectionUtils.isEmpty(chosenBackendIds)) {
-            List<Long> backendIds = systemInfoService.getBackendIds(true);
-            throw new DdlException("Failed to find enough host in all backends. need: " + replicationNum +
-                    ", Current alive backend is [" + Joiner.on(",").join(backendIds) + "]");
+        if (!CollectionUtils.isEmpty(chosenBackendIds)) {
```

Comment:
This is OK for creating table/partition. But for alter operation, there is a daemon scheduler to add/remove replicas on be to match the replication_num property. This check cannot cover the alter case.

Reply:
Do you mean the error message is inaccurate?

Reply:
Updated, PTAL